code/network/multi_ingame.cpp

```diff
@@ -88,6 +88,8 @@
 LOCAL	int	Ingame_ships_deleted = 0;
 //LOCAL	int	Ingame_ships_to_delete[MAX_SHIPS];
 
+SCP_vector<int> Ingame_ship_choices;
```

Comment:
This is going to be a pretty small vector but it's still good to clean up after once it's no longer needed. A `.clear()` call followed by a `.shrink_to_fit()` somewhere that's safe to call from both UI's once the selection screen closes.

I can't tell if `multi_ingame_select_close()` gets called when using the scripted interface, but if so that would be the place to put it.

Reply:
`multi_ingame_select_close()` isn't called because it only handles retail UI stuff. But I've added a clear and shrink_to_fit to it and the API close method.